test/over_react/util/class_names_test.dart

```diff
@@ -239,6 +239,65 @@ main() {
             }));
           });
         });
+
+        group('merge', () {
+          test('returns className merged from second builder instance', (){
+            ClassNameBuilder otherBuilder = new ClassNameBuilder();
+
+            builder
+              ..add('a');
+              
+            otherBuilder
+              ..add('b');
+            
+            builder.merge(otherBuilder);
+
+            expect(builder.toClassName(), equals('a b'));
+          });
+
+          test('returns blacklist merged from second builder instance', (){
+            ClassNameBuilder otherBuilder = new ClassNameBuilder();
+
+            builder
```

Comment:
Since these are only single setter usages and there's no use case for the returned instance of `ClassNameBuilder` that the cascade provides, it would be the same functionality and more terse to do:

```dart
builder.blacklist('a-blacklist');
otherBuilder.blacklist('b-blacklist');
```